song_rater/forms.py

```diff
@@ -0,0 +1,6 @@
+from django import forms
+
+
+class AddSongForm(forms.Form):
+    artist = forms.CharField(label='artist', max_length=100)
+    title = forms.CharField(label='title', max_length=100)
```

Comment:
OPTIONAL: It might be nice to use a model form here https://docs.djangoproject.com/en/3.1/topics/forms/modelforms/
that way you don't have to duplicate the `max_length` imformation between model and form

Reply:
I added a create view which handles all the logic in the background